sample/Evernote Business/server.py

```diff
@@ -0,0 +1,233 @@
+		user = user_store.getUser()
+		if not user.accounting.businessId:
+			return render_template("error.html", error_message="This account is not a part of a business.  Please click here <a href=\'/clear'>click here</a> to reset and try another account.")
+		else:
+			business_store = client.get_business_note_store()
+
+		#setup search
+		notebook_filter=NoteStoreTypes.NoteFilter()
+		result_spec = NoteStoreTypes.NotesMetadataResultSpec(includeTitle=True)
+
+		# Start seraching for notes in the last day
+		# if less than 5 results increase to 10, 100, etc. until more than 5 are found or 
+		# the number results between iterations are the same
+		search_results_len = None
+		while search_results_len<10:
+			#setup counter
+			try:
+				counter += 1
+			except NameError:
+				counter = 0
+			#define number of pervious days to search:
+			days_pervious_to_search = str(int(math.pow(10, counter)))
+			#Define search grammer (docs: https://dev.evernote.com/doc/articles/search_grammar.php)
+			notebook_filter.words = "created:day-"+days_pervious_to_search+" updated:day-"+days_pervious_to_search
+
+			#perform search
+			search_results = business_store.findNotesMetadata(notebook_filter,0 , 40000, result_spec)
+			
+			#break if you get the same number of results for a order of magnitude increase in the number of days
+			# (assume that is the total number of search results present)
+			if len(search_results.notes)==search_results_len and search_results_len != 0:
+				break
+
+			# if you search the last 10,000 days and there are not business notes
+			# assume there are no business notes and inform the user
+			if counter == 4 and search_results_len==0:
+				return render_template("error.html", error_message="This account has no business notes!  Please click please add and sync some business notes and refresh the page.")
+
+			search_results_len = len(search_results.notes)
+
+		notes_metadata = search_results.notes
+
+		auth_result = user_store.authenticateToBusiness()
+		business_shard_id = auth_result.user.shardId
+		business_user_id = auth_result.user.id
+		business_token = auth_result.authenticationToken
+
+		#return the list of recently created and edited business notes and their views and display them (with their links) to the user
+		for note in notes_metadata:
+			#Get the HTML contents of each note: evernote:///view/32687061/s12/ff36290-4c35-4938-ab73-c09ce4a83e1c/ff36290-4c35-4938-ab73-c09ce4a83e1c/
+			template_link = "evernote:///view/%s/%s/%s/%s/" % (business_user_id, business_shard_id, note.guid, note.guid)
+			#edit this template: Link to note "evernote:///view/[userId]/[shardId]/[noteGuid]/[noteGuid]/"
+
+			#create URL that allows the user to create a new note based on the template
+			in_app_link = "note/template/%s" % note.guid
+
+			#get the thumnail for each note
+			r=requests.post(EN_URL+"/shard/"+business_shard_id+"/thm/note/"+note.guid+".jpg", data={"auth":business_token})
```

Comment:
Small nit: be consistent in your style; always put a space on either side of a symbol like `=` or `+` - and wrap lines (i.e. to 90 characters) for readability.